src/FastConv/Convolver.cpp

```diff
@@ -0,0 +1,323 @@
+#include "Convolver.h"
+
+ConvolverInterface::ConvolverInterface() {
+    reset();
+}
+
+ConvolverInterface::~ConvolverInterface() {
+    reset();
+}
+
+Error_t ConvolverInterface::init(float* impulseResponse, int irLength, int blockLength) {
+    m_IR = impulseResponse;
+    m_IRLength = irLength;
+    m_blockLength = blockLength;
+    return Error_t::kNoError;
+}
+
+Error_t ConvolverInterface::reset() {
+    m_IRLength = 0;
+    m_blockLength = 0;
+    return Error_t::kNoError;
+}
+
+void ConvolverInterface::applyWetGain(float* output, int length) {
+    CVectorFloat::mulC_I(output, m_wetGain, length);
```

Comment:
What does CVectorFloat::mulC_I do? I  cannot even google it.

Reply:
I think that's provided in `Vector.h` that is written by Alex.